MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -240,6 +272,61 @@ - (MSPriority)priority {
   return MSPriorityHigh;
 }
 
+#pragma mark - MSLogManagerDelegate
```

Comment:
Also might be good for our future selves to add a comment saying: We had no choice than to implement the last chance storage/flush logic in the crash module even if it concerns logs from any services because we can't contact any other module while crashing.

If I am not mistaken ^^.